_schemaV2.graphql

```diff
@@ -15458,6 +15468,49 @@ type updateNotificationPreferencesMutationPayload {
   ): [NotificationPreference!]!
 }
 
+type UpdatePartnerArtworksMutationFailure {
+  mutationError: GravityMutationError
+}
+
+input UpdatePartnerArtworksMutationInput {
+  # Whether Artsy domestic shipping should be enabled
+  artsyShippingDomestic: Boolean
+
+  # Whether Artsy international shipping should be enabled
+  artsyShippingInternational: Boolean
+  clientMutationId: String
+
+  # ID of the partner
+  id: String!
+
+  # The partner location ID to assign
+  location: String
+}
+
+type UpdatePartnerArtworksMutationPayload {
+  clientMutationId: String
+  partnerArtworksOrError: UpdatePartnerArtworksMutationType
+}
+
+type UpdatePartnerArtworksMutationSuccess {
+  partnerArtworks: UpdatePartnerArtworksMutationSuccessDetails
+}
+
+type UpdatePartnerArtworksMutationSuccessDetails {
+  errors: PartnerArtworksError
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID likely used as a database ID.
+  internalID: ID!
```

Comment:
Is there a database ID returned from that PUT? (doesn't look like it based on https://github.com/artsy/gravity/blob/53acd88de5deb8b8c3ac7c84cc57cc0a31660618/spec/api/v1/partners_artworks_endpoint_spec.rb#L759)